Scripts/download-portal-item-data.swift

```diff
@@ -200,10 +260,19 @@ portalItems.forEach { (portalURLString, portalItems) in
     portalItems.forEach { (portalItem) in
         // Have we already downloaded the item?
         let filename = downloadedItems[portalItem.identifier] ?? portalItem.filename
-        if FileManager.default.fileExists(atPath: destinationURLProvider.makeURL(filename: filename).path) {
+        let tempURL = destinationURLProvider.makeURL(filename: filename)
+        let pathURL = tempURL.pathExtension == "zip" ? tempURL.deletingPathExtension() : tempURL
```

Comment:
Since paths and URLs are two different things, I would suggest renaming `pathURL` to `fileURL`.